src/app/Components/email-preview/email-preview.component.scss

```diff
@@ -5,4 +5,8 @@
 i {
   width: auto;
   height: 3px;
+}
+.email-preview {
+  height: 300px;
+  overflow: auto;
 }
```

Comment:
add an empty line at the end of any file
or use format from Prettier VSCode extension for helping with this "lint".